CHANGELOG.md

```diff
@@ -11,7 +20,7 @@ v7.16.0
 - Added `f-templates-loader` module which handles locating and registering shared templates.
 - Added `config.docs.excludeTemplateDirs` property to allow directories to be ignored when adding shared templates.
 
-### Updated
+### Changed
```

Comment:
I can't believe there isn't an `Updated`.  This part of my life has been a lie…

Reply:
I know right, once I noticed it I just couldn't allow it to stay though.. 😄